sdk/containerregistry/azure-containerregistry/azure/containerregistry/_container_registry_client.py

```diff
@@ -72,6 +90,15 @@ def list_repositories(self, **kwargs):
         :return: ItemPaged[str]
         :rtype: :class:`~azure.core.paging.ItemPaged`
         :raises: :class:`~azure.core.exceptions.ResourceNotFoundError`
+
+        .. admonition:: Example:
+
+            .. literalinclude:: ../samples/sample_delete_old_tags.py
```

Comment:
is there a reason you pointed to this sample rather than the `list_repositories` call in `sample_create_client.py`? As a user who wants to see how `list_repositories` is being used, I would probably think to look in `sample_create_client.py` (or some other all-encompassing sample or a separate `list_repositories` sample) rather than in the delete old tags sample.

Reply:
We _think_ deleting old tags is a good use case for the library so I point them there for the sample, but if this thought changes in UX studies so will the sample.